docs/reference/src/language/builtins/elements.md

```diff
@@ -216,6 +216,7 @@ or it will be mapped to a private unicode character. The mapping of these non-pr
 
 -   **`focus()`** Call this function to transfer keyboard focus to this `FocusScope`,
     to receive future [`KeyEvent`](structs.md#keyevent)s.
+-   **`clear-focus()`** Call this function to remove keyboard focus from this `FocusScope` if it currently has the focus.
```

Comment:
Maybe this should document that clear-focus doesn't set the focus to anything else and that no element have the focus afterwards.
Like what is going to be the next item to get the focus when pressing tab once this function is called?

Reply:
Sure, I can elaborate a little.